lib/rules/no-restricted-invocations.js

```diff
@@ -57,13 +57,22 @@ module.exports = class NoRestrictedInvocations extends Rule {
     let checkDenylist = (node) => {
       let denylist = this.config;
 
+      const name = this._getComponentOrHelperName(node);
+      if (!name) {
+        return;
+      }
+
+      const nameToDisplay = node.type === 'ElementNode' ? `<${node.tag} />` : `{{${name}}}`;
+
       for (let denylistItem of denylist) {
```

Comment:
We could also get rid of this outer for loop as well if we changed `parseConfig` to return a `Map` mapping invalid component/helper names to error messages instead of leaving it as a list.

Reply:
I actually tested a similar change of converting the `names` array to a set inside `parseConfig`, and that reduced the running time by 1-2 seconds for my very large application, but I decided against making that change for now due to the slight added complexity and negligible performance time benefit (theoretically, it could also cause a slight increase in memory usage). It would be neat if there was a way to pass the config into the rule as a set/map in the first place so we wouldn't have to convert it for every single file.

My experimental change:
```js
return config.map((item) => {
    if (typeof item === 'object') {
        return { names: new Set(item.names), message: item.message };
    } else {
        return item;
    }
});
```

Reply:
It would increase the complexity of `parseConfig`, but I think that would be more than offset by decreasing the complexity of `checkDenylist`.